include/roaring/roaring.h

```diff
@@ -904,85 +904,140 @@ typedef struct roaring_uint32_iterator_s {
 } roaring_uint32_iterator_t;
 
 /**
- * Initialize an iterator object that can be used to iterate through the
- * values. If there is a  value, then this iterator points to the first value
- * and `it->has_value` is true. The value is in `it->current_value`.
+ * Initialize an iterator object that can be used to iterate through the values.
+ * If there is a  value, then this iterator points to the first value and
+ * `it->has_value` is true. The value is in `it->current_value`.
  */
-void roaring_init_iterator(const roaring_bitmap_t *r,
+void roaring_iterator_init(const roaring_bitmap_t *r,
                            roaring_uint32_iterator_t *newit);
 
+/** DEPRECATED, use `roaring_iterator_init`. */
+CROARING_DEPRECATED inline void roaring_init_iterator(
```

Comment:
If these are just `inline`, they also need an `extern inline` in the `.c` file, to provide an implementation when e.g. optimizations are off, or an actual function implementation needs to live somewhere for some reason (the function's address is taken, etc).

Instead, these could be `static inline` functions, which means we don't need an extern implementation in the c file